ocfweb/docs/docs/services/web.md

```diff
@@ -11,6 +11,52 @@ where `user` is the account name.
 Group accounts may also request a [[virtual host|doc services/vhost]] for
 another domain (e.g., group.berkeley.edu).
 
+#### Off-site hosting
+
+One question that often comes up is if the OCF is able to host websites
+from services such as Squarespace, Wix, Weebly, Tumblr, etc. Unfortunately
+due to logistical considerations and University policy, the OCF is unable
+to do so. More specifically, the OCF cannot point  <group>.berkeley.edu
+domain names to off-campus hosting providers, but it can, however, host
+websites exported from such providers.
+
+Many website building services like Wix and Weebly make their money by
+using a proprietary service to allow users to design their websites, and
+subsequently charge users to host those sites. This means providing the
+servers, storage space, internet connection, and configurations to allow
+these sites to show up on the internet. Obviously, the OCF is unable to
+access this proprietary software and is therefore unable to host sites
+developed using them. However, the OCF is capable of hosting anything made
+using the free and open-source backends we support, including Wordpress,
+Django, Ruby on Rails, Node.js, PHP, and others. You may wish to consider
+building your website with those if you desire to host with the OCF.
```

Comment:
I like this section, maybe mention the ability to export from those providers though in case they are already using them? I would make sure to discourage exporting from their service and importing to the OCF though because that complicates setting up their site and probably wouldn't be as easy to maintain due to whatever obfuscation/plugins they use or other stuff like that (unless it's simpler, like migrating from Wordpress.com to Wordpress at the OCF, which we already have a docs section for).